lib/administrate/search.rb

```diff
@@ -19,9 +19,10 @@ def run
     private
 
     delegate :resource_class, to: :resolver
+    delegate :table_name, to: :resource_class
```

Comment:
I would use `resource_class.table_name` because you aren't' frequently changing data types as you chain and you don't expect `resource_class` to become possibly `nil` at any point.